components/speedreader/resources/panel/components/font-size-control/index.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react'
+
+import * as S from './style'
+import { FontSize } from '../../api/browser'
+
+enum ActionType {
+  Inc,
+  Dec
+}
+
+interface FontSizeControlProps {
+  currentSize: FontSize
+  onClick?: Function
+}
+
+function FontSizeControl (props: FontSizeControlProps) {
+  const updateSize = (action: ActionType) => {
+    const newSize = action === ActionType.Dec ? props.currentSize - 10 : props.currentSize + 10
+    if (newSize >= FontSize.MIN_VALUE && newSize <= FontSize.MAX_VALUE) {
+      props.onClick?.(newSize)
+      return
+    }
+    return props.onClick?.(props.currentSize)
+  }
+
+  return (
+    <S.Box>
+      <S.ButtonLeft
+        onClick={() => updateSize(ActionType.Dec)}
+      >
+        <svg width="24" height="24" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M19.01 11H4.99c-.533 0-.99.461-.99 1 0 .539.457 1 .99 1h14.096c.457-.077.914-.461.914-1 0-.539-.457-1-.99-1Z" fill="currentColor"/></svg>
```

Comment:
How about putting inline SVGs in a separate named component so that we can easily know what they are representing when looking at the code.

Reply:
done